### ISSUE TITLE
[$10]The 'submit' link appears grayed out visually but it is not announced as disabled by the screen reader.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3237]
**Describe the bug**

The 'submit' link appears grayed out visually but it is not announced as disabled by the screen reader.
ted labels.

**To Reproduce**

1. Go to the sample challenges page
2. Ensure that the screen reader is switched on. 
3. Navigate to the 'submit' link using the arrow keys. 
4. Notice that this link appears greyed out visually however it is not announced as disabled by the screen reader."

**Expected behavior**

The screen reader should announce the 'submit' link as disabled when it is not available for activation.

**Actual behavior**
The screen reader doesn't announce the 'submit' link as disabled even though it is visually greyed out.

**Additional context**

Type: Submission

Page: https://www.topcoder.com/challenges/30095347

Status: Not fixed, Ideally this could be made a button which is initially disabled, and becomes available once user registers.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099906 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099906 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099906 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
PR at https://github.com/topcoder-platform/topcoder-react-utils/pull/49

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099906<br/><br/>```This is an automated message for crazyk via Topcoder X```
